osu.Game/Screens/Play/MasterGameplayClockContainer.cs

```diff
@@ -152,10 +183,10 @@ public override void Seek(double time)
         /// </summary>
         public void Skip()
         {
-            if (GameplayClock.CurrentTime > gameplayStartTime - MINIMUM_SKIP_TIME)
+            if (GameplayClock.CurrentTime > latestGameplayStartTime - MINIMUM_SKIP_TIME)
```

Comment:
What's the difference in referring to `latestGameplayStartTime` here instead of `StartTime`?

Reply:
As mentioned in my last comment, they are completely different variables with different meanings and usages. Can you have a read over the code which uses `latestGameplayStartTime` and potentially suggest a better name for it?